docs/Developers/Development_Setup.rst

```diff
@@ -91,28 +100,28 @@ master branch from and all of its dependencies with pip using
     (coala-venv)$ cd coala-bears
     (coala-venv)$ pip3 install -e .
 
-Then you can install a repository-backed version of the repository you would
-like to modify using
+-Then you can install a repository-backed version of the repository you would
+ like to modify using
 
 ::
 
     (coala-venv)$ pip3 install -e <path/to/clone>
 
-You will then be able to edit the repository and have the changes take effect
-in your virtualenv immediately. You will also be able to use pip to manage
-your installation of the package should you need to install from a different
-source in the future.
+-You will then be able to edit the repository and have the changes take effect
+ in your virtualenv immediately. You will also be able to use pip to manage
+ your installation of the package should you need to install from a different
+ source in the future.
 
 
 Building Documentation
 ----------------------
 
-You should run this command before trying to build the documentation:
+-You should run this command before trying to build the documentation:
 
 ::
 
     (coala-venv)$ pip3 install -r docs-requirements.txt
 
-Once you have done so, you can build the documentation by entering the docs
-directory and running ``make``. The documentation on the coala website is in
-the ``coala`` (not ``coala-bears``) repository.
+-Once you have done so, you can build the documentation by entering the docs
+ directory and running ``make``. The documentation on the coala website is in
+ the ``coala`` (not ``coala-bears``) repository.
```

Comment:
@NiklasMM will you merge if I changed these , or others will appear saying something else , I am losing hope now.